emails/templates/emails/direct_email_header.html

```diff
@@ -159,7 +159,11 @@
             }
         }
     </style>
-
+{% if pretext_ftlmsg_id %}
+    <div style="display:none !important; color: #ffffff; font-size: 0px; line-height: 0; max-height: 0px; max-width: 0px; opacity: 0; overflow: hidden; mso-hide: all;">
```

Comment:
What is the point of this section?

Reply:
To implement "preheader text" as shown in the figma. I had to look it up too: https://www.activecampaign.com/blog/email-preheader